webpack.config.analyzer.js

```diff
@@ -0,0 +1,6 @@
+import {BundleAnalyzerPlugin} from 'webpack-bundle-analyzer';
+import config from './webpack.config.dev';
+
+config.plugins.push(new BundleAnalyzerPlugin());
```

Comment:
Would it make sense to run the analyzer against the production webpack.config instead of dev? Since that's what will actually be deployed I'm thinking analyzing prod would be most relevant...